scripts/cluster

```diff
@@ -158,9 +158,15 @@ function initRootchain() {
 function startServerFromBinary() {
   if [ "$1" == "write-logs" ]; then
     echo "Writing validators logs to the files..."
-    ./polygon-edge server --data-dir ./test-chain-1 --chain genesis.json \
-      --grpc-address :10000 --libp2p :30301 --jsonrpc :10002 --relayer \
-      --num-block-confirmations 2 --seal --log-level DEBUG 2>&1 | tee ./validator-1.log &
+    if [ "$1" == "polybft" ]; then
```

Comment:
This is not correct, because we are sending `write-logs` flag value to this function (so "$1" contains value for it and not the consensus type).

Figure out if possible setting some global variable denoting consensus type being used or checking the correct flag value.

Note that cluster script can be run in a following ways
`scripts/cluster polybft [write-logs]`, meaning that write-logs is optional parameter.